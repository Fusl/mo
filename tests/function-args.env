name=Willy
MO_ALLOW_FUNCTION_ARGUMENTS=true

pipeTo() {
    cat | "$1"
}

testArgs() {
    printf "%d" "$#"

    # Display all arguments
    printf " %q" ${@+"$@"}
}
