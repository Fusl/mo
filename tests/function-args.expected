No args: 0 '' - done
One arg: 1 one - done
Getting name in a string: 1 The\ name\ is\ Willy - done
Reverse this: edcba
